src/ducks/search/SearchHeader.jsx

```diff
@@ -24,17 +26,30 @@ const SearchHeader = ({
 }) => {
   const { isMobile } = useBreakpoints()
   const { t } = useI18n()
+  const {
+    isSelectionModeEnabled,
+    isSelectionModeActive,
+    setIsSelectionModeActive
+  } = useSelectionContext()
 
   const handleReset = inputNode => {
     setSearch('')
+    if (isSelectionModeEnabled) setIsSelectionModeActive(false)
     inputNode.focus()
   }
 
   const handleChange = useCallback(
     ev => {
       setSearch(ev.target.value)
+      if (isSelectionModeEnabled && results.length <= 0)
```

Comment:
How it's possible to have a negative value ?

Reply:
Je sais pas, c'était au cas ou, je me suis dis que ça coutait pas plus cher... Mais si ça pose plus de question qu'autre chose un `=== 0` devrait suffire oui ;)

Reply:
Plus clair le `===` en tout cas 😄